cockatrice/src/window_main.cpp

```diff
@@ -455,7 +455,7 @@ void MainWindow::registerError(Response::ResponseCode r, QString reasonStr, quin
             QMessageBox::critical(this, tr("Registration denied"), tr("The email address provider used during registration has been blacklisted for use on this server."));
             break;
         case Response::RespTooManyRequests:
-            QMessageBox::critical(this, tr("Registration denied"), tr("Too many registration attempts, please try again later or contact the server operator for further details."));
+            QMessageBox::critical(this, tr("Registration denied"), tr("It appears you are attempting to register a new account on this server yet you already have an account registered and the server restricts the number of accounts a user can register.  Please contact the server operator for further assistance or to obtain your credential information."));
```

Comment:
How concerned with security are you?  I was trying to be as descriptive as possible without giving away to much information.  If we put something specific like "already registered with that email" then there is the potential for someone to hammer away at the registration process throwing email accounts at it and seeing if they get a response saying that is valid against an account.